src/Connection.php

```diff
@@ -47,11 +47,15 @@ class Connection extends EventEmitter
      */
     public $parser;
 
-    public function __construct(LoopInterface $loop, array $connectOptions = array())
+    public function __construct(LoopInterface $loop, array $connectOptions = array(), ConnectorInterface $connector = null)
```

Comment:
It seems the `ConnectorInterface` comes from `React\Socket` namespace, but it is not imported yet?